src/test/shell/testenv.sh

```diff
@@ -465,6 +465,22 @@ http_archive(
 EOF
 }
 
+function add_rules_java_to_workspace() {
```

Comment:
Why does rules_java need to be added?

Reply:
That raises quite a few questions wrt. dependency management, but all of them are out of scope for this pull request and let's try to fix problems one at a time. Approved!